README.md

```diff
@@ -48,6 +53,31 @@ OPENSSL_PREFIX="/usr/local/Cellar/openssl@1.1/1.1.1q" carton install
 You may need to check `/usr/local/Cellar/openssl@1.1` to find the latest
 installed path on your system.
 
+#### cmark
+
+If you like, you can install the `cmark` C libraries via Homebrew:
+
+```bash
+brew install cmark
```

Comment:
I think this may be better left as part of the `brew install carton` command. The elaboration added in this PR doesn't change anything about how cmark is installed, just how it's used. And it should generally be installed before running carton.

Linking against the version specific paths shouldn't be needed and will be fragile. Upgrades could lead to breaking your build. So I'd rather list it as a fallback in case the normal path didn't work rather than the expected path.